safe-global/web-core
# Fix: filter pending txs by safe address

## What it solves

Resolves #1817

## How this PR fixes it

When we check if there are pending transactions, we need to filter all pending transactions by chainId and safeAddress.
So I've added `safeAddress` to the pending transaction type.
This wasn't required before because we were always selecting pending txs by id (taken from the queue or history).

## How to test it
* In a 1/X Safe, create and immediately execute a tx
* Attempt to create and execute another tx in the same Safe – it won't let you execute it while the 1st one is still pending.
* In another 1/X Safe, attempt to create and execute a tx – you should be able to do so regardless of the other Safe's state
